Userland/Applets/Audio/main.cpp

```diff
@@ -237,12 +237,13 @@ class AudioWidget final : public GUI::Widget {
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    TRY(Core::System::pledge("stdio recvfd sendfd rpath wpath cpath unix thread"));
+    TRY(Core::System::pledge("stdio recvfd sendfd rpath wpath cpath unix thread proc"));
 
     auto app = TRY(GUI::Application::try_create(arguments));
     Config::pledge_domain("AudioApplet");
-    TRY(Core::System::unveil("/tmp/user/%uid/portal/audio", "rw"));
+    TRY(Core::System::unveil("/tmp/session/%sid/portal/audio", "rw"));
```

Comment:
These being in a separate commit means that these commits aren't atomic?
We should avoid that.

Reply:
I'll squash these back in when i decide what I'm doing with the `/etc/passwd` problem. do you have any opinon on that btw?

Reply:
done